src/api/app/helpers/webui/patchinfo_helper.rb

```diff
@@ -4,4 +4,27 @@ def patchinfo_bread_crumb(*args)
     args.insert(0, link_to(@package, package_show_path(project: @project, package: @package)))
     project_bread_crumb(*args)
   end
+
+  def patchinfo_header(patchinfo, package_names)
+    list = package_names.to_sentence
+    text = "#{patchinfo.category} update for #{truncate(list, length: 60)}"
+    capture_haml do
+      header_title(patchinfo, text)
+      header_subtitle(patchinfo.summary, text, list)
+    end
+  end
+
+  private
+
+  def header_title(patchinfo, text)
+    haml_tag(:h3, class: 'mb-0') do
+      haml_concat(patchinfo.summary.presence || text)
+      haml_tag(:span, 'blocked', class: 'badge badge-blocked') if patchinfo.block
+      haml_tag(:span, patchinfo.rating, class: "badge badge-rating #{patchinfo.rating}")
```

Comment:
Adding a title like `Rating of this patchinfo` would make it obvious what this is about.

I wonder if rating could be renamed to Severity. But this is out of scope.